code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -118,11 +126,22 @@
 	SSair.stop_processing_machine(src)
 	SSair.rebuild_queue -= src
 
-	if(pipe_vision_img)
-		qdel(pipe_vision_img)
+	QDEL_NULL(pipe_vision_img)
+	QDEL_NULL(cap_overlay)
 
 	return ..()
-	//return QDEL_HINT_FINDREFERENCE
+
+/**
+ * Handler for `COMSIG_OBJ_HIDE`, connects only if `hide` is set to `TRUE`. Calls `update_cap_visuals` on pipe and its connected nodes
+ */
+/obj/machinery/atmospherics/proc/on_hide(datum/_source, _underfloor_accessibility)
```

Comment:
```suggestion
/obj/machinery/atmospherics/proc/on_hide(datum/source, underfloor_accessibility)
```

there's no reason to change the name of the argument to be obfuscated for `datum/source` (in fact I would prefer if you didn't do it), and I think it's weird to have it for `underfloor_accessibility` as well. we have to assume that people will be smart about using those two arguments in the future, and there's no real pressing reason to imply that it's a bad idea to use them imo (but if you have other reasons do let me know)